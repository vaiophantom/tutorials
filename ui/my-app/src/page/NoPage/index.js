import Header from "../../components/Header";

const NoPage = () => {
  const handleSearch = (type, value) => {};
  return (
    <div>
      <Header handleSearch={handleSearch} />
      <div className="body">NoPage</div>
    </div>
  );
};
export default NoPage;
