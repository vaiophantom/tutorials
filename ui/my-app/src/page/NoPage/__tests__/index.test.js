import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import NoPage from '../';

it('renders Checkout', async () => {
 const {container} = render(<NoPage />, {wrapper: BrowserRouter});
 expect(container).toMatchSnapshot();
});
