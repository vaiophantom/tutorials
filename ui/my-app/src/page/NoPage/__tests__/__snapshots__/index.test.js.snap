// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Checkout 1`] = `
<div>
  <div>
    <div
      class="header"
    >
      <img
        class="logo"
        height="100"
        src="logo.png"
        width="200"
      />
      <div
        class="search"
      >
        <div
          class="input-group"
          style="background-color: white; align-items: center;"
        >
          <button
            aria-expanded="false"
            class="dropdown-toggle btn btn-outline-primary"
            type="button"
          >
            name
          </button>
          <div
            aria-labelledby=""
            class="dropdown-menu"
            x-placement="bottom-start"
          >
            <a
              class="dropdown-item"
              data-rr-ui-dropdown-item=""
              href="#"
              role="button"
              tabindex="0"
            >
              name
            </a>
            <a
              class="dropdown-item"
              data-rr-ui-dropdown-item=""
              href="#"
              role="button"
              tabindex="0"
            >
              branch
            </a>
            <a
              class="dropdown-item"
              data-rr-ui-dropdown-item=""
              href="#"
              role="button"
              tabindex="0"
            >
              category
            </a>
            <a
              class="dropdown-item"
              data-rr-ui-dropdown-item=""
              href="#"
              role="button"
              tabindex="0"
            >
              color
            </a>
          </div>
          <input
            aria-describedby="basic-addon3"
            class="form-control"
            placeholder="Search products based on category name, product name, price, branch..."
            value=""
          />
          <img
            class=""
            height="30"
            src="search.jpeg"
            width="30"
          />
        </div>
      </div>
      <img
        class="cart"
        height="40"
        src="cart.png"
        width="40"
      />
    </div>
    <div
      class="body"
    >
      NoPage
    </div>
  </div>
</div>
`;
